### ISSUE TITLE
[Crash] Crash when linking to Nature's Aura Everlasting Spring

### ISSUE THREAD

**Issue by MuteTiefling** [SuperMartijn642/Entangled1.12#64]
**Version Info**
- Minecraft, 1.19.2
- Entangled, entangled-1.3.14-forge-mc1.19.2

**Steps to Reproduce**
Place an Everlasting Spring
Place an Entangled Block
Link the Spring to the Entangled Block

**Crash report (~/logs/latest.log)**
https://gist.github.com/MuteTiefling/34159b6b1c2bf39eeded08d7e7d8d422

**Comment by SuperMartijn642** [comment]
Seems to be caused by Nature's Aura. I have created a pull request to fix it here: https://github.com/Ellpeck/NaturesAura/pull/321

I have also added extra checks when rendering blocks. Any errors will now simply be reported in the logs and the respective blocks will no longer be rendered until the game is restarted, rather than crashing the game whenever an error occurs.
This is available in version 1.3.15.

For now I'll close the issue as it is up to Nature's Aura to fix it.
Thank you for reporting the issue!
